### DISCUSSION TITLE
Logo Permissions

### DISCUSSION THREAD

**Discussion by Eutropios** [helix-editor/helix#10958] [Q&A] [answered]
Hi there,

I'm looking to make a [Wikipedia article](https://en.wikipedia.org/wiki/Draft:Helix_(text_editor)) on the Helix editor, and I'd like to include the logo in the article. However, I'm unsure what the logo is licensed under. If the image is not CC-BY-SA or some other permissive/copyleft license, then I would need the permission of the creator to upload it. If you don't want your image on Wikipedia, then that's okay too 😄.

Thanks!

**Comment by kirawi** [comment, marked as answer]
It is licensed under the MPL 2.0 license

**Reply by Eutropios** [reply]
Thanks for the quick response! I'll go ahead and [upload it to Wikimedia Commons](https://commons.wikimedia.org/wiki/File:Helix_editor.svg) so it can be added to the article. If at any time you'd like it to be taken down, feel free to open a [Deletion request](https://commons.wikimedia.org/wiki/Commons:Deletion_requests)!

**Reply by David-Else** [reply]
> **Attribution**: MPL 2.0 requires that you provide appropriate credit to the original creator.

Who should it be attributed to?

**Reply by Eutropios** [reply]
I have attributed it to Jake Langford, as they created it during the [discussion](https://github.com/helix-editor/helix/issues/283#issuecomment-1332089825). If there are any corrections you'd like to make to the metadata of the file in the Commons (such as creation date, attribution, and so forth), you're more than welcome to [edit it](https://commons.wikimedia.org/wiki/File:Helix_editor.svg). Wikimedia is a collective effort after all. 😄
